dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESContentletAPIImpl.java

```diff
@@ -3993,6 +3995,17 @@ private void internalUnpublish(final Contentlet contentlet, final User user, fin
 
         this.canLock(contentlet, user);
 
+        if (contentlet.isHTMLPage()) {
+            final Optional<Experiment> runningExperiment = APILocator.getExperimentsAPI().getRunningExperiments().stream()
```

Comment:
shouldn't be a method such as getExperimentPerPage

Reply:
done https://github.com/dotCMS/core/pull/25760/files#diff-8bf1865184fd59c5c64882df72bd0c71a403a544777830acf250bbf54ea84d90R238